superset-frontend/src/components/ListView/CardCollection.tsx

```diff
@@ -36,19 +37,36 @@ const CardContainer = styled.div`
     ${({ theme }) => theme.gridUnit * 4}px;
 `;
 
+const CardWrapper = styled.div`
+  border: 2px solid transparent;
+  &.card-selected {
+    border: 2px solid ${({ theme }) => theme.colors.primary.base};
+  }
+`;
+
 export default function CardCollection({
-  renderCard,
+  bulkSelectEnabled,
+  loading,
   prepareRow,
+  renderCard,
   rows,
-  loading,
-}: Props) {
+}: CardCollectionProps) {
   return (
     <CardContainer>
       {rows.map(row => {
         if (!renderCard) return null;
         prepareRow(row);
         return (
-          <div key={row.id}>{renderCard({ ...row.original, loading })}</div>
+          <CardWrapper
+            className={
+              row.isSelected && bulkSelectEnabled ? 'card-selected' : ''
+            }
+            key={row.id}
+            onClick={() => row.toggleRowSelected()}
```

Comment:
We should probably check to see if `bulkSelectEnabled` before firing this. Like this, I think it's possible to select cards even if bulkSelect is not enabled.